docs/api/chinese-postman/api-reference.md

```diff
@@ -0,0 +1,46 @@
+# Chinese Postman service API reference
```

Comment:
we should slap a big fat (BETA) on this top level lable and then put a note below that says something like: While this API may be fully functional we are currently treating it as beta and may still make breaking changes to the API in the future.

Reply:
would be also good to add the information about the two failure cases that you added to the PR discription into the text here. i think we are totally fine to iterate on this rather than get it perfect on the first go.

Reply:
Added.